models/dataset_test.go

```diff
@@ -83,8 +84,8 @@ func TestCreateDataset(t *testing.T) {
 	Convey("Return with error when the request body contains the correct fields but of the wrong type", t, func() {
 		b, err := json.Marshal(badInputData)
 		if err != nil {
-			log.ErrorC("Failed to marshal test data into bytes", err, nil)
-			os.Exit(1)
+			t.Logf("failed to marshal test data into bytes, error: %v", err)
```

Comment:
I think it would be more concise to use `So(err, ShouldBeNil)`

Reply:
See response on previous comment